src/Client.ts

```diff
@@ -17,6 +18,7 @@ export default class Client {
     public basketLine!: BasketLine;
     public subscriptions!: Subscriptions;
     public subscriptionAddons!: SubscriptionAddons;
+    public codes!: Codes;
```

Comment:
You need to instantiate this resource below in the `loadResources` method.

Reply:
There's also the tests that need updating please @dawid-whirli

Reply:
Done. @yasgreen93 I've also added tests for resources you created for subscriptions.